uwblueprint/social-diversity-for-children
# Neel/updating page styles for mobile

### Notion ticket link

[Start Mobile Responsive Designs for static pages](https://www.notion.so/uwblueprintexecs/Start-Mobile-Responsive-Designs-for-static-pages-71b40c3d94f1448ab3fde0ca1d376fb8)



### Implementation description

- Implemented the useBreakPointValue hook to determine device type and added logic to various components to appropriately style based on the recognized device type

### Steps to test

1. Right click on the webpage in the Chrome web browser, and click "Inspect"
2. Change the device type to mobile in the device toolbar
3. Verify that the webpages match the Figma designs when in the mobile view

#### Videos for reference during testing:
**Document Upload Page:**

https://user-images.githubusercontent.com/23081661/141712591-4185d423-448e-4fdf-8726-ec7503493e62.mov


**Home Page:**

https://user-images.githubusercontent.com/23081661/141713281-5ad89202-d19d-4dc2-b43d-c4f00559a4c5.mov


**Login Page:**

https://user-images.githubusercontent.com/23081661/141712667-fff30149-1748-4b97-979b-62906db651b7.mov


### What should reviewers focus on?

- Ensure that the webpages match the Figma designs

### Checklist
